artifactory-enterprise/azuredeploy.json

```diff
@@ -0,0 +1,871 @@
+      "metadata":  {
+        "description": "Provide your SSL Certificate key"
+      }
+    },
+    "certificateDomain": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "artifactory",
+      "metadata":  {
+        "description": "Provide your Certificate Domain Name. For e.g jfrog.team for certificate with *.jfrog.team"
+      }
+    },
+    "artifactoryServerName": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "artifactory",
+      "metadata":  {
+        "description": "Provide artifactory server name to be used in Nginx. e.g artifactory for artifactory.jfrog.team"
+      }
+    },
+    "extraJavaOptions": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "-server -Xms2g -Xmx12g -Xss256k -XX:+UseG1GC -XX:OnOutOfMemoryError=\\\"kill -9 %p\\\"",
+      "metadata":  {
+        "description": "Setting Java Memory Parameters for Artifactory. Learn about system requirements for Artifactory https://www.jfrog.com/confluence/display/RTF/System+Requirements#SystemRequirements-RecommendedHardware."
+      }
+    },
+    "DB_Admin_User": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Database Admin user name"
+      }
+    },
+    "DB_Admin_Password": {
+      "type": "securestring",
+      "minLength": 1,
+      "metadata": {
+        "description": "Database Admin password"
+      }
+    },
+    "DB_Name": {
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Database name"
+      }
+    },
+    "DB_Edition": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "Basic",
+      "allowedValues": [
+        "Basic",
+        "Standard",
+        "Premium"
+      ],
+      "metadata": {
+        "description": "Database Edition"
+      }
+    },
+    "DB_Location": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Database location. Default to resource group location if blank."
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/artifactory-enterprise/"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      },
+      "defaultValue": ""
+    }
+  },
+  "variables": {
+    "clusterLocation": "[if(equals(parameters('clusterLocation'), ''), resourceGroup().location, parameters('clusterLocation'))]",
```

Comment:
remove this an use [resourceGroup().location] as the defaultValue on the parameter